frontend/src/components/common/Table.tsx

```diff
@@ -0,0 +1,244 @@
+import { Paper } from '@mui/material';
+import { useTheme } from '@mui/styles';
+import {
+  MaterialReactTable,
```

Comment:
Can you please test this on mobile? (if you haven't already)

Reply:
didn't test it too thoroughly but it looks good to me, adapts to small widths and handles touch

Reply:
Great, thanks.